plugins/coral-plugin-love/client/LoveButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Icon} from 'coral-ui';
+import styles from './styles.css';
+import {I18n} from 'coral-framework';
+import translations from './translations.json';
+import withReaction from 'coral-framework/hocs/withReaction';
+const lang = new I18n(translations);
+
+class LoveButton extends React.Component {
+  handleClick = () => {
+    const {
+      postReaction,
+      deleteReaction,
+      showSignInDialog,
+      reactionSummary
+    } = this.props;
+    const {root: {me}, comment} = this.props;
+
+    // If the current user does not exist, trigger sign in dialog.
+    if (!me) {
```

Comment:
This essentially exposes the result of our query from the graph. We could provide more abstracted props that does not relate to our graph, but on the other hand our graph is fully exposed on the server side anyway, so I don't really see a point. (high prio for stabilising our graph schema)